src/components/View/Servizio/ServizioSchemaOrg.jsx

```diff
@@ -0,0 +1,120 @@
+import { toPublicURL, isInternalURL } from '@plone/volto/helpers';
+import { useIntl } from 'react-intl';
+import { SiteProperty } from 'volto-site-settings';
+import {
+  SchemaOrg,
+  SchemaOrgUtils,
+  richTextHasContent,
+  getSiteProperty,
+} from 'io-sanita-theme/helpers';
+
+const ServizioSchemaOrg = ({ content }) => {
+  const intl = useIntl();
+
+  let siteTitle = SiteProperty({
+    property: 'site_title',
+    getValue: true,
+    defaultTitle: getSiteProperty('siteTitle', intl.locale),
+  });
+
+  const schemaOrg = {
+    '@type': 'GovernmentService',
+    name: content.title,
+    serviceType: content.parent.title,
+    serviceOperator: {
+      '@type': 'GovernmentOrganization',
+      name: siteTitle,
+    },
+    areaServed: {
+      '@type': 'AdministrativeArea',
+      name: richTextHasContent(content.copertura_geografica)
+        ? SchemaOrgUtils.fieldDataToPlainText(content.copertura_geografica)
+        : siteTitle,
+    },
+    availableChannel: {
+      '@type': 'ServiceChannel',
+      name: 'Dove rivolgersi',
+      availableLanguage: {
+        '@type': 'Language',
+        name: 'Italian',
+        alternateName: 'it',
+      },
+    },
+  };
+
+  let description = [];
+  if (content.description) {
+    description.push(content.description);
+  }
+  if (richTextHasContent(content.descrizione_estesa)) {
+    description.push(
+      SchemaOrgUtils.fieldDataToPlainText(content.descrizione_estesa),
+    );
+  }
+  if (description.length > 0) {
+    schemaOrg.description = description.join('. ');
+  }
+
+  // a chi si rivolge
+  if (richTextHasContent(content.a_chi_si_rivolge)) {
+    schemaOrg.audience = {
+      '@type': 'Audience',
+      audienceType: SchemaOrgUtils.fieldDataToPlainText(
+        content.a_chi_si_rivolge,
+      ),
+    };
+  }
+
+  if (content.prenota_online_link) {
+    schemaOrg.availableChannel.serviceUrl = isInternalURL(
+      content.prenota_online_link,
+    )
+      ? toPublicURL(content.prenota_online_link)
+      : content.prenota_online_link;
+  } else {
+    schemaOrg.availableChannel.serviceUrl = toPublicURL(content['@id']);
+  }
```

Comment:
```suggestion
 }
```
questo non lo metterei, perchè nello schema org
la description per questo campo è:

serviceUrl | URL | The website to access the service.
-- | -- | --


[serviceUrl](https://schema.org/serviceUrl)	[URL](https://schema.org/URL)	The website to access the service.

L'url della pagina corrente da solo informazioni generali sul servizio, non è il link per accedere al servizio